code/modules/atmospherics/machinery/components/binary_devices/circulator.dm

```diff
@@ -125,3 +170,16 @@
 	..()
 	pixel_x = 0
 	pixel_y = 0
+
+/obj/machinery/atmospherics/components/binary/circulator/verb/circulator_flip()
+	set name = "Flip"
+	set category = "Object"
+	set src in oview(1)
+

```

Comment:
You need to check if the user is capable of doing this, so ghosts and dead can't flip it around. I'd prefer if this was part of the rotation component but right now there's no simple way to stop the actual rotation so this is fine for now.

Reply:
I make ishuman check.